src/components/map/ha-map.ts

```diff
@@ -247,19 +251,21 @@ export class HaMap extends ReactiveElement {
 
         // DRAW point
         this._mapPaths.push(
-          Leaflet!.circleMarker(path.points[pointIndex], {
-            radius: 3,
-            color: path.color || darkPrimaryColor,
-            opacity,
-            fillOpacity: opacity,
-            interactive: false,
-          })
+          Leaflet!
+            .circleMarker(path.points[pointIndex].point, {
+              radius: 3,
+              color: path.color || darkPrimaryColor,
+              opacity,
+              fillOpacity: opacity,
+              interactive: true,
+            })
+            .bindPopup(path.points[pointIndex].tooltip)
```

Comment:
Can we use hover instead of click?

Reply:
Looks like `bindTooltip` instead of `bindPopup` does a hover instead of a click. I will try that.